CountingWords.rb

```diff
@@ -0,0 +1,6 @@
+freq = Hash.new(0)
```

Comment:
This is something that I learned later on that you can initialize a hash with default values. Kind of cool. Well done.

Anytime you build up an empty collection, iteratively build it up then utilize it as input for something else, you should always consider `.reduce ` or a similar method as possibly a solution. in this case you're using `.scan` but reduce would have worked as well as `.each_with_object`.

http://batsov.com/articles/2013/12/04/using-rubys-each-with-object/